src/components/common/error/errorType.ts

```diff
@@ -0,0 +1,5 @@
+import { AppDetails } from '../../v2/appDetails/appDetails.type'
+
+export interface ErrorBarType {
+    appDetails: AppDetails
+}
```

Comment:
Can we keep this in the common-types file as there's only one type?

@vivek-devtron What are your thoughts on this?

Reply:
We can keep this for now as we'll be moving related types to this file in later PRs.